.github/workflows/cleanup-test.yml

```diff
@@ -48,6 +48,12 @@ jobs:
 
           echo credentials=$(realpath ~/gcp_sa_cred.json) >> $GITHUB_OUTPUT
 
+      - name: Download tools for openshift test cleaner
+        run: |
+          wget https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/latest-4.6/openshift-client-linux.tar.gz -O openshift.tar.gz -q
```

Comment:
4.6 is very old, also I am a bit concerned about keeping this up-to-date. Can't we inline this in the script itself?

Reply:
This is from the upgrade test, which still works. We use a limited set of OC commands. But we can bump them to the latest one

Reply:
I think we can, but this is only needed for this script. You can't launch it locally anyway